pom.xml

```diff
@@ -62,10 +62,8 @@
     <jenkins.baseline>2.462</jenkins.baseline>
     <!-- TODO Replace with the standard jenkins.baseline references after LTS requires Java 17  -->
     <!-- <jenkins.version>${jenkins.baseline}.1</jenkins.version> -->
```

Comment:
https://github.com/jenkinsci/bom/pull/3677 IIUC

Reply:
BOM will release Friday, then I plan to release the git client plugin 6.1.0 using that new plugin BOM release.

Reply:
Or maybe it is better to release git client plugin 6.1.0 before the weekly BOM so that it is available sooner to BOM consumers

Reply:
The more I think about it, the more I think it is better to release this today then include it in the plugin BOM release tomorrow